base/univention-config-registry/python/univention-config-registry

```diff
@@ -54,10 +54,7 @@ def convertToUtf8(arguments):
 		try:
 			argument = argument.decode('UTF-8').encode('UTF-8')
 		except UnicodeError:
-			try:
-				argument = argument.decode(localeencoding).encode('UTF-8')
-			except UnicodeError:
```

Comment:
I don't thinks so. "argument" is set right from the start, otherwise argument.decode() would throw a NameError.

Reply:
Of cours, you are right. Sorry for the noise. I have no idea why I oversaw this. I dropped the commit.